corehq/apps/app_manager/feature_support.py

```diff
@@ -68,3 +68,10 @@ def enable_group_in_field_list(self):
         version 2.16 or higher
         """
         return self._require_minimum_version('2.16')
+
+    @property
```

Comment:
@sravfeyn You thought there was a better way for feature support, this is where Danny pointed me.